creativecloud/features/firefly/firefly-masonry.css

```diff
@@ -0,0 +1,477 @@
+
+/* stylelint-disable property-no-vendor-prefix */
+/* stylelint-disable value-no-vendor-prefix */
+/* stylelint-disable selector-no-vendor-prefix */
+.ff-masonry .mobile-only,
+.ff-masonry .grid-layout {
+  display: none;
+}
+
+.masonry-prompttext::-moz-placeholder,
+.masonry-prompttext.light::-moz-placeholder {
+  font-style: italic;
+}
+
+.masonry-prompttext::placeholder,
+.masonry-prompttext.light::placeholder {
+  font-style: italic;
+}
+
+.masonry-promptbar > button:hover {
+  cursor: pointer;
+}
+
+.preload {
+  display: none;
+}
+
+.ff-masonry .grid-container { 
+  display: grid;
+  width: 635px;
+  height: 408px;
+  grid-template-columns: repeat(12, 1fr);
+  column-gap: var(--spacing-s);
+  row-gap: var(--spacing-xs);
+  justify-content: center;
+  overflow: hidden;
+}
+
+.image-container {
+  position: relative;
+}
+
+.con-button { 
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
Can you make these relative i.e. to be applied to .con-button only inside the container. else this will impact .con-button anywhere on the page with firefly-masonry

Reply:
done